apis/telemetry/v1alpha1/telemetry_types.go

```diff
@@ -4,14 +4,26 @@ package v1alpha1
 
 type TelemetrySpec struct {
 	Prometheus *PrometheusTelemetrySpec `json:"prometheus,omitempty"`
+	Mcac       *McacTelemetrySpec       `json:"mcac,omitempty"`
```

Comment:
Nit: `MCAC` according to https://github.com/golang/go/wiki/CodeReviewComments. (scroll up slightly, the anchor link for initialisms is broken.)

Reply:
Also, just double checking whether `mcac` is right for the yaml, my gut feeling is that this is the right way to do it, but just drawing your attention to it, as I couldn't find a style guide for CRDs - I think they should follow whatever is preferred for JSON schemas.